wordpress-mobile/WordPress-iOS
# Update SVProgress HUD in order to have proper multitask mode support.

Fixes #6651 

To test:
 - Start the app on the iPad, and switch to an multitask mode
 - Go to any of the editor
 - Create a post
 - Tap on Save or Publish
 - Check that the HUD shows in the right position and right appearance

Needs review: @astralbodies
